src/System.Net.Security/tests/FunctionalTests/NegotiateStreamKerberosTest.cs

```diff
@@ -34,13 +34,13 @@ public static IEnumerable<object[]> GoodCredentialsData
                 // Anonymous (with domain name).
                 yield return new object[] { new NetworkCredential(
                     Configuration.Security.ActiveDirectoryUserName,
-                    null,
+                    (string)null,
                     Configuration.Security.ActiveDirectoryName) };
                 
                 // Anonymous (without domain).
                 yield return new object[] { new NetworkCredential(
                     Configuration.Security.ActiveDirectoryUserName,
-                    null,
+                    (string)null,
```

Comment:
Hah, no worries! Will look if I find any other .NET Core code that used null on NetworkCredential ctor :)